packages/babel-generator/src/generators/typescript.ts

```diff
@@ -693,6 +693,7 @@ export function TSNamespaceExportDeclaration(
   this.word("namespace");
   this.space();
   this.print(node.id, node);
+  this.semicolon();
```

Comment:
This is the actual fix. The rest is just for consistency with the rest of `@babel/generator`, where we use `this.semicolon()` since it also does some state cleanup.